src/components/com_tjcertificate/administrator/models/forms/filter_certificates.xml

```diff
@@ -41,6 +41,14 @@
 			<option value="">COM_TJCERTIFICATE_CERTIFICATE_FILTER_ISSUED_USER_SELECT</option>
 		</field> -->
 
+		<field name="agency_id"
+			type="agencies"
+			filter="safehtml"
```

Comment:
the filter is not correct...it should be integer...check and fix

Reply:
done